stacks/style.css

```diff
@@ -0,0 +1,16 @@
+/*
+Theme Name: Stacks
+Theme URI: 
+Author: Automattic
+Author URI: 
+Description: Stack your story
+Requires at least: 5.8
+Tested up to: 5.9
+Requires PHP: 5.7
+Version: 0.0.1
+License: GNU General Public License v2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+Template: 
```

Comment:
if it's not a child theme I think this can go away. Again, if this is from CBT, we should maybe file an issue so that it's not included when we are not creating a child theme

Reply:
Done